tests/pruners_tests/test_percentile.py

```diff
@@ -90,6 +91,7 @@ def test_25_percentile_pruner_intermediate_values(
     assert trial.should_prune()
 
 
+@pytest.mark.filterwarnings("ignore::RuntimeWarning")
 def test_25_percentile_pruner_intermediate_values_nan() -> None:
```

Comment:
[Note] The decorator is applied to the test function since this test focuses on `nan`.